homeassistant/components/filter/sensor.py

```diff
@@ -132,7 +134,12 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Required(CONF_ENTITY_ID): cv.entity_id,
+        vol.Required(CONF_ENTITY_ID): vol.All(
+            cv.entity_id,
```

Comment:
I think we can drop this since it's included in `cv.entity_domain`.

https://github.com/home-assistant/core/blob/ad634a393b352b8f8b3000debad386cef0cce5b4/homeassistant/helpers/config_validation.py#L315